lib/rails_api_logger/request_log.rb

```diff
@@ -6,7 +6,7 @@ class RequestLog < ActiveRecord::Base
 
   belongs_to :loggable, optional: true, polymorphic: true
 
-  scope :failed, -> { where.not(response_code: 200..299) }
+  scope :failed, -> { where(response_code: 400..599).or(where.not(ended_at: nil).where(response_code: nil)) }
```

Comment:
```suggestion
  scope :failed, -> { where(response_code: 400..599).or(where.not(ended_at: nil, response_code: nil)) }
```

Reply:
Your suggestion alters the logic. The meaning should be "where ended but no response"

Reply:
Oh right!